pkg/gardenlet/controller/shoot/care/add.go

```diff
@@ -53,7 +56,17 @@ func (r *Reconciler) AddToManager(mgr manager.Manager, gardenCluster cluster.Clu
 				r.EventHandler(),
 				r.ShootPredicate()),
 		).
-		Complete(r)
+		Build(r)
+	if err != nil {
+		return err
+	}
+
+	return c.Watch(
```

Comment:
Could be rewritten with `builder.Watches()`

Reply:
Done. Replaced with build.WatchesRawSource.